docs/designing-apis/validation-style-guide.md

```diff
@@ -0,0 +1,21 @@
+---
+tags: []
+---
+
+# Customize Style & Validation Rules
```

Comment:
This one needs to be updated for Studio, with Next references removed. We should also update it to 'Validation & Linting', since there is no customization possible today like there was in Next.

Reply:
updated to reflect differences